Source/Hurl.BrowserSelector/Windows/MainWindow.xaml.cs

```diff
@@ -21,26 +21,7 @@ public MainWindow()
 
         InitializeComponent();
 
-        var osbuild = Environment.OSVersion.Version.Build;
-        var backtype = settings.AppSettings?.BackgroundType;
-
-        if (settings.AppSettings?.NoWhiteBorder == true) WindowBorder.BorderThickness = new Thickness(0);
-        if (osbuild < 22000) WindowBorder.CornerRadius = new CornerRadius(0);
-
-        if (backtype == "acrylic" && osbuild >= 22523)
-        {
-            WindowBackdropType = WindowBackdropType.Acrylic;
-        }
-        else if (backtype == "none" || osbuild < 22000)
-        {
-            WindowBackdropType = WindowBackdropType.None;
-            var brush = Color.FromRgb(150, 50, 50);
-            Background = new SolidColorBrush(brush);
-        }
-        else
-        {
-            WindowBackdropType = WindowBackdropType.Mica;
-        }
```

Comment:
I still prefer to have acrylic

Might have to revert this back unfortunately, while also trying to support SystemThemeWatcher

Reply:
Mica or Acrylic is both okay.
```xaml
WindowBackdropType="Mica"
```
Should be the place for this.